integration/terraform/ec2/win/variables.tf

```diff
@@ -0,0 +1,49 @@
+variable "ec2_instance_type" {
+  type = string
+  default = "t3a.xlarge"
+}
+
+variable "key_name" {
+  type = string
+  default = "cwagent-integ-test-key"
+}
+
+variable "iam_instance_profile" {
+  type = string
+  default = "CloudWatchAgentServerRole"
+}
+
+variable "vpc_security_group_ids" {
+  type = list(string)
+  default = ["sg-013585129c1f92bf0"]
+}
+
+variable "region" {
+  type = string
+  default = "us-west-2"
+}
+
+variable "ami" {
+  type = string
+  default = ""
+}
+
+variable "ssh_key" {
+  type = string
+  default = ""
+}
+
+variable "user" {
```

Comment:
where is this variable used?

Reply:
I should not have copied code even my own lol

Reply:
windows user will also be Administrator for win

Reply:
My fault for not looking more closely at the terraform